assets/manifest.json

```diff
@@ -22,6 +22,7 @@
   },
   "options_ui": {
     "page": "options.html",
+    "open_in_tab": true,
```

Comment:
This value is `false` by default. If set to `true`, the options page will open in a normal browser tab, rather than being integrated into the browser's add-ons manager.

This change is needed as a work around for this bug, where the dialog is jumping up and down  https://bugs.chromium.org/p/chromium/issues/detail?id=904942#c1